### PR TITLE
Correção de tradução e acréscimo de detalhes técnicos;

### PR DESCRIPTION
**Pull request by paiva-thiago**
Correções de tradução e acréscimos de detalhes de linguagem.

- [X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] None of my changes are plagiarized from another source without proper attribution.
- [X] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

File: guide/portuguese/javascript/html-dom/index.md [hunk 1]
```diff
@@ -1,16 +1,14 @@
 ---
-title: HTML Dom
-localeTitle: HTML Dom
+title: HTML DOM (Document Object Model)
```
Comment (CleoAguiar) [inline guide/portuguese/javascript/html-dom/index.md]: Please verify the Title section in the [guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md#title): Article titles should be as short, concise, and as to-the-point as possible.
```suggestion
title: HTML DOM (Document Object Model)
```

**Review by CleoAguiar — CHANGES_REQUESTED** [review]

File: guide/portuguese/javascript/html-dom/index.md (hunk 1 above)
Comment (RandellDawson) [reply guide/portuguese/javascript/html-dom/index.md]: @CleoAguiar  Actually, the title should be the same as the filename.  If everything else looks good here, then we can merge this one.

@paiva-thiago Please change the title back to the original `HTML Dom`

Comment (paiva-thiago) [reply guide/portuguese/javascript/html-dom/index.md]: @RandellDawson I made a commit with this change. Do I need to do a new pull request for that?

Comment (paiva-thiago) [reply guide/portuguese/javascript/html-dom/index.md]: @RandellDawson Done!

**Review by RandellDawson — APPROVED** [review]

**Pull request merged**